langchain/llms/gpt4all.py

```diff
@@ -122,14 +131,41 @@ def _llama_param_names() -> Set[str]:
             "embedding",
         }
 
+    @staticmethod
+    def _gptj_param_names() -> Set[str]:
+        """Get the identifying parameters."""
+        return set()
+
+    @staticmethod
+    def _get_param_names(backend: str) -> Set[str]:
+        if backend == "llama":
+            return GPT4All._llama_param_names()
+        else:
+            return GPT4All._gptj_param_names()
+
+    def _get_default_params(self) -> Dict[str, Any]:
```

Comment:
could this still be a property (and still be called `_default_params`)?

Reply:
Thank you for the comment, changed.